src/lv_draw/lv_draw_label.c

```diff
@@ -60,6 +60,13 @@ void lv_draw_label(const lv_area_t * coords, const lv_area_t * mask, const lv_st
 {
     const lv_font_t * font = style->text.font;
     lv_coord_t w;
+
+    /*No need to waste processor time if string is empty*/
+    if (txt[0] == '\0')
```

Comment:
Considering that the bulk of the calculation is done in a loop, this should be a safe change.

@kisvegabor Please look this over just to make sure there aren't any side effects that people may rely on that were caused by the prior behavior.

Reply:
This patch introduces changes into `lv_draw_label()` behavior. But I assume that this is rather bugfix, than breaking backwards compatibility. I've observed following behavior: if string like "\0A\0" is passed as txt param to `lv_draw_label()`  then two-line label is drawn (first line is empty and second line contains 'A' symbol). I believe this comes form `line_end` local variable (https://github.com/littlevgl/lvgl/blob/dev-6.1/src/lv_draw/lv_draw_label.c#L108) which will be always greater than `line_start` even in case if zero-length string  is passed (because of https://github.com/littlevgl/lvgl/blob/dev-6.1/src/lv_misc/lv_txt.c#L309). If this is not a bug and someone relies on this behavior tell me and I'll rework my PR. If you want this fix to go into master branch as a separate PR, let me know and I'll submit one more PR and rework structure of this one.

Reply:
That's really a bug. Thank you for the fix!